src/Components/Blazor/Blazor/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -66,7 +66,8 @@ public Task AddComponentAsync(Type componentType, string domElementSelector)
             // (otherwise the logged exception will come from UpdateDisplayAsync instead of here)
             // When implementing support for out-of-process runtimes, we'll need to call this
             // asynchronously and ensure we surface any exceptions correctly.
-            ((IJSInProcessRuntime)JSRuntime.Current).Invoke<object>(
+
+            WebAssemblyJSRuntime.Current.Invoke<object>(
```

Comment:
Should we get this from services or constructor?

Reply:
Spoke about this offline. It's kinda iffy because we really need an instance of `MonoWebAssemblyJSRuntime` to be passed in to the ctor. It doesn't help with unit testing much and we can clean it up in the future if it turns out to be a problem.